Assets/Scripts/UI/TwitchLeaderboard.cs

```diff
@@ -31,50 +27,74 @@ private void Start()
             return;
         }
 
-        float delay = 0.6f;
-        int index = 0;
+        if (leaderboard.Success)
+        {
+            promptTransform.FindChild("Retry").gameObject.SetActive(false);
+        }
 
-        foreach (Leaderboard.LeaderboardEntry entry in leaderboard.GetSortedEntries(maximumRowCount))
+        statsTable = Instantiate<TwitchLeaderboardStats>(twitchLeaderboardStatsPrefab);
+        statsTable.leaderboard = leaderboard;
+        
+        if (leaderboard.Count > 0)
         {
-            TwitchLeaderboardRow row = Instantiate<TwitchLeaderboardRow>(index < specialRows.Length ? specialRows[index] : normalRow);
-            row.position = index + 1;
-            row.leaderboardEntry = entry;
-            row.delay = delay;
-            row.transform.SetParent(tableTransform, false);
+            mainTable = Instantiate<TwitchLeaderboardTable>(twitchLeaderboardTablePrefab);
+            mainTable.leaderboard = leaderboard;
+        }
 
-            _instancedRows.Add(row);
+        if (leaderboard.SoloCount > 0)
+        {
+            soloTable = Instantiate<TwitchLeaderboardTableSolo>(twitchLeaderboardTableSoloPrefab);
+            soloTable.leaderboard = leaderboard;
+            transform.FindChild("UI").FindChild("Mask").gameObject.SetActive(true);
```

Comment:
Ah ok. Covering it up with a black box is fine really (not overly concerned by the amount of overdraw concerned).